Resources/public/fonts/icomoon.svg

```diff
@@ -64,7 +64,29 @@
 <glyph unicode="&#xe904;" glyph-name="List_view" d="M96 928h148.219c35.346 0 64-28.654 64-64v-146.286c0-35.346-28.654-64-64-64h-148.219c-35.346 0-64 28.654-64 64v146.286c0 35.346 28.654 64 64 64zM96 585.143h148.219c35.346 0 64-28.654 64-64v-146.286c0-35.346-28.654-64-64-64h-148.219c-35.346 0-64 28.654-64 64v146.286c0 35.346 28.654 64 64 64zM96 242.286h148.219c35.346 0 64-28.654 64-64v-146.286c0-35.346-28.654-64-64-64h-148.219c-35.346 0-64 28.654-64 64v146.286c0 35.346 28.654 64 64 64zM458.055 928h464.164c35.346 0 64-28.654 64-64v-146.286c0-35.346-28.654-64-64-64h-464.164c-35.346 0-64 28.654-64 64v146.286c0 35.346 28.654 64 64 64zM458.055 585.143h464.164c35.346 0 64-28.654 64-64v-146.286c0-35.346-28.654-64-64-64h-464.164c-35.346 0-64 28.654-64 64v146.286c0 35.346 28.654 64 64 64zM458.055 242.286h464.164c35.346 0 64-28.654 64-64v-146.286c0-35.346-28.654-64-64-64h-464.164c-35.346 0-64 28.654-64 64v146.286c0 35.346 28.654 64 64 64z" />
 <glyph unicode="&#xe905;" glyph-name="Grid_view" d="M440.246 551.031c0-17.274-14.003-31.277-31.277-31.277h-313.692c-17.274 0-31.277 14.003-31.277 31.277v313.692c0 17.274 14.003 31.277 31.277 31.277h313.692c17.274 0 31.277-14.003 31.277-31.277v-313.692zM960 864.723c0 17.274-14.004 31.277-31.277 31.277h-313.692c-17.273 0-31.277-14.003-31.277-31.277v-313.692c0-17.273 14.004-31.277 31.277-31.277h313.692c17.273 0 31.277 14.004 31.277 31.277v313.692zM440.246 344.969c0 17.273-14.003 31.277-31.277 31.277h-313.692c-17.274 0-31.277-14.004-31.277-31.277v-313.692c0-17.273 14.003-31.277 31.277-31.277h313.692c17.274 0 31.277 14.004 31.277 31.277v313.692zM960 344.969c0 17.273-14.004 31.277-31.277 31.277h-313.692c-17.273 0-31.277-14.004-31.277-31.277v-313.692c0-17.273 14.004-31.277 31.277-31.277h313.692c17.273 0 31.277 14.004 31.277 31.277v313.692z" />
 <glyph unicode="&#xe92c;" glyph-name="copy" d="M640 704v256h-448l-192-192v-576h384v-256h640v768h-384zM192 869.49v-101.49h-101.49l101.49 101.49zM64 256v448h192v192h320v-192l-192-192v-256h-320zM576 613.49v-101.49h-101.49l101.49 101.49zM960 0h-512v448h192v192h320v-640z" />
+<glyph unicode="&#xe92f;" glyph-name="folder" d="M448 832l128-128h448v-704h-1024v832z" />
 <glyph unicode="&#xe934;" glyph-name="move" d="M576 704l-128 128h-448v-832h1024v704h-448zM512 480l224-224h-160v-256h-128v256h-160l224 224z" />
 <glyph unicode="&#xe989;" glyph-name="enlarge" d="M1024 960h-416l160-160-192-192 96-96 192 192 160-160zM1024-64v416l-160-160-192 192-96-96 192-192-160-160zM0-64h416l-160 160 192 192-96 96-192-192-160 160zM0 960v-416l160 160 192-192 96 96-192 192 160 160z" />
 <glyph unicode="&#xea10;" glyph-name="checkmark" d="M864 832l-480-480-224 224-160-160 384-384 640 640z" />
+<glyph unicode="" glyph-name="checkmark" d="M864 832l-480-480-224 224-160-160 384-384 640 640z" />
```

Comment:
I think you can try to remove this line, don't you ? The glyph with glyphname "checkmark is already here line 90... And unicode seems better...